test/gtest/uct/ib/test_ib_md.cc

```diff
@@ -322,6 +319,10 @@ UCS_TEST_SKIP_COND_P(test_ib_md, mt_fail,
     void *start, *mid, *last;
     uct_mem_h memh;
 
+    if (!has_ksm()) {
```

Comment:
Do we need to check UCT_MD_FLAG_INVALIDATE_RMA?

Reply:
i think we don't actually need it for MT registration